.buildkite/autogenerate_pipeline.py

```diff
@@ -116,6 +116,13 @@ def _set_platform(self, platform):
                 platform = 'arm'
             self.step_config['agents']['platform'] = f"{platform}.metal"
 
+    def _set_hypervisor(self, hypervisor):
+        """ Set hypervisor if given in the json input. """
+        supported_hypervisors = ['kvm', 'mshv']
```

Comment:
Right now we don't have a tag for kvm. We can leave it just in case we want to extend it in the future though.